rikaichan.js

```diff
@@ -308,22 +308,30 @@ var rcxMain = {
 	inlineEnable: function(tab, mode) {
 		if (!this.dict) {
 			//	var time = (new Date()).getTime();
-			if (!this.loadDictionary()) return;
+			var loadProm = this.loadDictionary();
+
+			if (loadProm == null)
```

Comment:
It'd be better to have return on the same line or add braces.  it's very easy to add another line to an uncontained if without realizing it doesn't work.